apps/src/applab/designMode.js

```diff
@@ -859,7 +859,7 @@ designMode.renderToggleRow = function (screenIds) {
   var designToggleRow = document.getElementById('designToggleRow');
   if (designToggleRow) {
     React.render(
-      React.createElement(DesignToggleRow, {
+      React.createElement(PlaySpaceHeader, {
```

Comment:
Tried this out.  Our build system handles it just fine, but our linter doesn't expect to find XML elements in .js files.